tensorflow/python/lib/io/file_io_test.py

```diff
@@ -195,6 +196,20 @@ def testGetMatchingFilesWhenParentDirContainsParantheses(self):
     self.assertItemsEqual(
         file_io.get_matching_files(glob_pattern), expected_match)
 
+  @unittest.expectedFailure
+  def testGetMatchingFilesWhenContainsGlob(self):
+    # https://git.io/JR8jL (#35489)
```

Comment:
Can we make this be `expectedFailure(link to github issue)`?

Reply:
I don't it has a reason like argument
https://docs.python.org/3/library/unittest.html